bin/mail/2016-12-05-hoc-global-is-here/generate-contacts

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# All teachers (based on the teacher queries from 2016-11-03-hoc-global-site-refresh)
+# And non-teachers from
+#   VolunteerEngineerSubmission2015, VolunteerEngineerSubmission2016, and Petition non-educator
+
+queries = {
+  teachers: '
+    kind_s: HocSignup2016 ||
+    kind_s: HocHardwarePrizes2014 ||
+    kind_s: HocHardwarePrizes2015 ||
+    kind_s: HocSignup2014 ||
+    kind_s: HocSignup2015 ||
+    kind_s: HocSurvey2014 ||
+    kind_s: HocSurvey2015 ||
+    kind_s: CSEdWeekEvent2013 ||
+    (
+      kind_s: user && role_s: teacher
+    ) || (
+      kind_s: Petition && role_s: educator
+    ) ||
+    kind_s: BringToSchool2013 ||
+    kind_s: ClassSubmission ||
+    kind_s: DistrictPartnerSubmission ||
+    kind_s: HelpUs2013 ||
+    kind_s: K5OnlineProfessionalDevelopmentPostSurvey ||
+    kind_s: K5ProfessionalDevelopmentSurvey ||
+    kind_s: ProfessionalDevelopmentWorkshop ||
+    kind_s: ProfessionalDevelopmentWorkshopSignup ||
+    kind_s: StudentNomination ||
+    kind_s: TeacherNomination
+  ',
+
+  non_teachers: '
+    kind_s: VolunteerEngineerSubmission2015 ||
+    kind_s: VolunteerEngineerSubmission2016 ||
```

Comment:
This doesn't exist, AFAIK. The 2016 volunteers are included in the 2015 volunteers.